pgjdbc/src/main/java/com/yugabyte/ysql/ClusterAwareLoadBalancer.java

```diff
@@ -243,7 +265,20 @@ public synchronized boolean refresh(Connection conn) throws SQLException {
       return false;
     }
     lastServerListFetchTime = currTime;
-    unreachableHosts.clear();
+    long now = System.currentTimeMillis() / 1000;
+    long failedHostTTL = Long.getLong("failed-host-ttl-seconds", 5);
```

Comment:
Please put a comment about why hard coded to 5 seconds. Better to put this in a constant and explain this.